RosettaCore/src/main/java/com/hubspot/rosetta/Rosetta.java

```diff
@@ -32,11 +32,13 @@ public static void setMapper(ObjectMapper mapper) {
     INSTANCE.set(mapper);
   }
 
-  public static ObjectMapper cloneAndCustomize(ObjectMapper mapper) {
+  public static ObjectMapper cloneAndCustomize(ObjectMapper mapper, List<Module> modules) {
```

Comment:
This is a breaking API change. Could we leave a `cloneAndCustomize(ObjectMapper mapper)` method which delegates to the new one?